scripts/tests.sh

```diff
@@ -0,0 +1,27 @@
+# CI testing script
+
+sudo apt-get update && sudo apt-get install python3-venv
```

Comment:
Either, we run everything inside a docker image on CI and thus don't need to use `venv` (and actually shouldn't - see for example https://stackoverflow.com/questions/48561981/activate-python-virtualenv-in-dockerfile) or we do it on localhost and thus should do the venv preparation outside the script.

Reply:
You are right - fixed